src/pages/attestering/Attestering.tsx

```diff
@@ -0,0 +1,242 @@
+    const dispatch = useAppDispatch();
+
+    const behandling = sak.behandlinger.find((x) => x.id === behandlingId);
+    if (!behandling) {
+        return <AlertStripe type="feil">Fant ikke behandlingsid</AlertStripe>;
+    }
+    const formik = useFormik<FormData>({
+        initialValues: {},
+        onSubmit: (values) => {
+            const { beslutning } = values;
+
+            if (beslutning) {
+                dispatch(
+                    sakSlice.startAttestering({
+                        sakId: sak.id,
+                        behandlingId: behandling.id,
+                    })
+                );
+                return;
+            }
+            // TODO: Implementera vurdert på nytt når det implementeras i bakover
+        },
+        validationSchema: yup.object<FormData>({
+            beslutning: yup.boolean().required(),
+            grunn: yup.mixed<keyof typeof Grunn>().when('beslutning', {
+                is: false,
+                then: yup.mixed<keyof typeof Grunn>().oneOf([Grunn.FEIL_I_BEREGNING, Grunn.FEIL_I_PERIODE]).required(),
+            }),
+
+            begrunnelse: yup.string(),
+        }),
+        validateOnChange: hasSubmitted,
+    });
+
+    const { errors } = formik;
+    if (
+        behandling.vilkårsvurderinger &&
+        (behandling.status === Behandlingsstatus.TIL_ATTESTERING ||
+            behandling.status === Behandlingsstatus.ATTESTERT ||
+            behandling.status === Behandlingsstatus.AVSLÅTT)
+    ) {
+        return (
+            <div>
+                <div className={styles.vedtakContainer}>
+                    <Innholdstittel>Vedtak</Innholdstittel>
+                    <div>
+                        {behandling.status === Behandlingsstatus.TIL_ATTESTERING && (
+                            <AlertStripeSuksess>{Behandlingsstatus.SIMULERT}</AlertStripeSuksess>
+                        )}
+                        {behandling.status === Behandlingsstatus.AVSLÅTT && (
+                            <AlertStripeFeil>{Behandlingsstatus.AVSLÅTT}</AlertStripeFeil>
+                        )}
+                    </div>
+                    <div>
+                        <VilkårsOppsummering behandling={behandling} sakId={sak.id} />
+                    </div>
+                    <div>
+                        <VisDersomSimulert sak={sak} behandling={behandling} />
+                    </div>
+                    <div>
+                        <Innholdstittel>Vis brev kladd</Innholdstittel>
+                        <Lenke
+                            href={'#'}
+                            onClick={() =>
+                                fetchBrev(sak.id, behandlingId).then((res) => {
+                                    if (res.status === 'ok') window.open(URL.createObjectURL(res.data));
+                                })
+                            }
+                        >
+                            test
+                        </Lenke>
```

Comment:
Vi må huske å fikse på dette begge plasser. Vi skal _ikke_ bruke lenke som ikke er lenke og vi burde ha en spinner eller noe som viser progresjon.
Passer ikke helt inn i PR-en her, så vi kan ta det for seg selv.

Reply:
Okay, noterar ned det då så att vi husker det! 👍